.travis.yml

```diff
@@ -127,6 +127,9 @@ install:
         travis_retry sudo apt-get -qq -y install libegl1-mesa;
       fi;
 
+    # Install IPython
+    - pip -q install "ipython[notebook]" 
```

Comment:
and I think Travis should be able to use `conda` as well, which should be faster